.github/workflows/ci.yaml

```diff
@@ -14,13 +31,64 @@ jobs:
   latest-sdk-versions:
     runs-on: 'ubuntu-latest'
     outputs:
-      go_latest: '1.28.1'
-      typescript_latest: '1.10.3'
-      java_latest: '1.25.0'
-      python_latest: '1.7.0'
-      csharp_latest: '1.2.0'
+      go_latest: ${{ steps.get_release.go_latest }}
+      typescript_latest: ${{ steps.get_release.typescript_latest }}
+      java_latest: ${{ steps.get_release.java_latest }}
+      python_latest: ${{ steps.get_release.python_latest }}
+      csharp_latest: ${{ steps.get_release.csharp_latest }}
     steps:
-      - run: 'echo noop'
+      - name: Get the latest release version
+        id: get_release
+        run: |
```

Comment:
I wonder, since this is our own Go application here, if we should just make this a command on the features repo. Meaning you could run `go run . latest-sdk-version --lang go` (or maybe show all without `--lang` or maybe some kind of `sdk-info` command and version is just one of the values, or whatever no specific approach required). That way not only this GH workflow can use it, but anyone can use the command (and Go is a better programming language than bash, heh). Sorry I did not think of this earlier.

Reply:
Good idea, might as well build this into the features tool itself

Reply:
Thanks! Yeah, now anyone (including GHA) will be able to get this info